### PR TITLE
Rename *_test.rb to test_*.rb, revealing and fixing a failing test.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Note that `*_test.rb` files are not automatically run as part of the test suite, whereas `test_*.rb` files are.

### CODE REVIEW

File: lib/analyze_hoc_activity_helper.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 #
 # Helper methods for the analyze_hoc_activity cron job.
 #
+require File.expand_path('../../pegasus/src/env', __FILE__)
```
Comment (aoby) [inline lib/analyze_hoc_activity_helper.rb:5]: Is this related?

Comment (ashercodeorg) [inline lib/analyze_hoc_activity_helper.rb:5]: Yes. Without it the test fails, not having the src_dir method defined.

**Comment by aoby** [comment]
lgtm. Obviously run rake test from shared/ and make sure they all run and pass :)

**Comment by ashercodeorg** [comment]
Already manually ran bundle exec ruby -Itest on all the touched files. This clued me in to the failure. :)

**Comment by aoby** [comment]
Also be sure to actually run `rake test` and make sure all the tests are executed, since that was the original problem.
